sdflasher.sh

```diff
@@ -45,11 +45,11 @@ if [[ "$CONFIRM" == "Y" ]]; then
   echo "Flashing out $LATEST_VERSION to $RDISK"
   set -x
   if [[ "$LATEST_VERSION" == *zip  ]]; then
-    unzip -p "$RELEASEPATH/$LATEST_VERSION" | dd bs=1m of=$RDISK
+    unzip -p "$RELEASEPATH/$LATEST_VERSION" | dd bs=32M of=$RDISK
```

Comment:
IIRC `dd` is different on OS X than on Linux, and `32M` should be `32m` for some obscure reason.